src/Illuminate/Support/Collection.php

```diff
@@ -1266,6 +1266,25 @@ public function toBase()
         return new self($this);
     }
 
+    public function __get($name)
+    {
+        $nameToMethod = [
+            'where' => 'filter',
+            'unless' => 'reject',
+            'do' => 'each',
+            'extract' => 'map',
+            'sum' => 'sum',
+            'inOrderOf' => 'sortBy',
+            'inReverseOrderOf' => 'sortByDesc',
+        ];
```

Comment:
Yeah I've seen that afterwards. Too bad I liked the aliases you made...
But I've seen too that the merge and the update were fast and uncontrolled ! ^^"